amy/autoemails/actions.py

```diff
@@ -488,6 +491,8 @@ def check(event: Event):  # type: ignore
             #             Instructor Training
             and event.administrator
             and event.administrator.domain != "carpentries.org"
+            # 2023-09-20: also exclude CLDT
+            and event.administrator.domain != "carpentries.org/community-lessons/"
```

Comment:
I'm afraid these relate to Action classes, which are pickled and stored in Redis. We may not be able to update the already scheduled emails.

Does this matter for the already scheduled emails?

Reply:
It shouldn't matter for already scheduled, as this domain can't be selected as an administrator until after we release this PR.

Do we have to take any extra steps to update these classes in Redis during deployment?

Reply:
Everything scheduling these actions after the release will use the new conditions.